tests/unit_tests/test_margin_calculation.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append('.')
+from models.helper.MarginHelper import calculate_margin
+
+
+def test_calculate_negative_margin_on_binance_when_coin_price_over_1():
+    # this test using BNB-USDT as market
+    buy_fee = 0.04
+    buy_filled = 44.70642
+    buy_price = 667.26
+    buy_size = 0.067
+    debug = False
+    sell_fee = 0.0
+    sell_percent = 100
+    sell_price = 590.34
+    sell_taker_fee = 0.1
+    exchange = 'binance'
+
+    actual_margin, actual_profit, actual_sell_fee = calculate_margin(buy_size, buy_filled, buy_price, buy_fee,
+                                                                     sell_percent, sell_price, sell_fee, sell_taker_fee,
+                                                                     debug, exchange)
+    assert round(actual_margin, 2) == round(-11.538287612503474, 2)
```

Comment:
My calculation according to new margin calculation function is not same of the expected numbers.